Patches/MorrowRim/MorrowRim_KwamaCE.xml

```diff
@@ -0,0 +1,335 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+		<li>MorrowRim</li>
+		<li>MorrowRim - Just the Animals</li>
+		</mods>
+
+		<match Class="PatchOperationSequence">
+			<operations>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>/Defs/ThingDef[defName="MorrowRim_KwamaScrib"]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>QuadrupedLow</bodyShape>
```

Comment:
Serpentine may be appropriate, since it's a larval form.

Reply:
Kwama scrib resemble regular insectoids and aren't worm like.
If you are referring to another kwama forager, they are too fat and broad to be serpentine in shape.

Reply:
My mistake--I was referring to the forager, which should be serpentine. QuadrupedLow and Serpentine have the exact same width values, the only difference being their height. In this case, Serpentine is more suited.

Reply:
Forager is actually much taller than the scrib due to how thick it is lorewise. It really shouldn't be shorter than other creatures.

Reply:
It is? This is what I came up with for images, with the forager being far left: https://external-preview.redd.it/dfX3F-rvKgLSD1O76DChFKHTwZJe3qjyhm0lJk0LkGg.jpg?auto=webp&s=2dae003d51c511f974a1e2ed480070d59c1b096d

Reply:
Yea, I don't really know, for the changes on small shitty animals, feels free to change it on your own.